modules/040-control-plane-manager/docs/internal/ETCD_RECOVERY.md

```diff
@@ -460,7 +460,7 @@ On the affected node:
 1. Add master role to affected node.
```

Comment:
Hmm. This is not the `master` role anymore.

Reply:
Ok, it seems that there is a lot te refactor with the work `master`...

Reply:
Yup, and we still have `master` for our node group. I don't want to mix it in this PR. Do we need this renaming for deckhouse internals?

Reply:
In this particular case, I think yes. Because role in this context is the value of the node-role labels, that was changed.